<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TP3_GestionEtuClass</class>
 <widget class="QMainWindow" name="TP3_GestionEtuClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1031</width>
    <height>602</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TP3_GestionEtu</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBox_AddStudent">
         <property name="minimumSize">
          <size>
           <width>690</width>
           <height>75</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>75</height>
          </size>
         </property>
         <property name="title">
          <string>Ajout d'un étudiant</string>
         </property>
         <widget class="QPushButton" name="pushButton_addStudent">
          <property name="geometry">
           <rect>
            <x>560</x>
            <y>40</y>
            <width>121</width>
            <height>24</height>
           </rect>
          </property>
          <property name="text">
           <string>Ajout</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_add_number">
          <property name="geometry">
           <rect>
            <x>12</x>
            <y>40</y>
            <width>121</width>
            <height>22</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="label">
          <property name="geometry">
           <rect>
            <x>18</x>
            <y>20</y>
            <width>61</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>Numéro</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_add_firstname">
          <property name="geometry">
           <rect>
            <x>144</x>
            <y>40</y>
            <width>121</width>
            <height>22</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="label_2">
          <property name="geometry">
           <rect>
            <x>150</x>
            <y>20</y>
            <width>61</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>Prénom</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_add_lastname">
          <property name="geometry">
           <rect>
            <x>274</x>
            <y>40</y>
            <width>121</width>
            <height>22</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="label_3">
          <property name="geometry">
           <rect>
            <x>280</x>
            <y>20</y>
            <width>61</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>Nom</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_4">
          <property name="geometry">
           <rect>
            <x>400</x>
            <y>20</y>
            <width>61</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>Bac</string>
          </property>
         </widget>
         <widget class="QComboBox" name="lineEdit_add_BAC">
          <property name="geometry">
           <rect>
            <x>400</x>
            <y>40</y>
            <width>69</width>
            <height>22</height>
           </rect>
          </property>
         </widget>
         <widget class="QComboBox" name="lineEdit_add_department">
          <property name="geometry">
           <rect>
            <x>480</x>
            <y>40</y>
            <width>69</width>
            <height>22</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="label_5">
          <property name="geometry">
           <rect>
            <x>480</x>
            <y>20</y>
            <width>91</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>Département</string>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_DeleteStudent">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>75</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>75</height>
          </size>
         </property>
         <property name="title">
          <string>Suppression d'un étudiant</string>
         </property>
         <widget class="QPushButton" name="pushButton_delete_number">
          <property name="geometry">
           <rect>
            <x>560</x>
            <y>40</y>
            <width>121</width>
            <height>24</height>
           </rect>
          </property>
          <property name="text">
           <string>Supprimer</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_remove_number">
          <property name="geometry">
           <rect>
            <x>14</x>
            <y>40</y>
            <width>121</width>
            <height>22</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="label_8">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>20</y>
            <width>61</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>Numéro</string>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_Graphs" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QGroupBox" name="groupBox_department">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>250</height>
             </size>
            </property>
            <property name="title">
             <string>Départements d'origine</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_bacs">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>250</height>
             </size>
            </property>
            <property name="title">
             <string>Bacs d'origine</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="title">
       <string>Liste des étudiants</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="pushButton_darkMode">
         <property name="text">
          <string>Changer de thème</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="listWidget">
         <property name="selectionMode">
          <enum>QAbstractItemView::SelectionMode::ExtendedSelection</enum>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_delete_list">
         <property name="text">
          <string>Supprimer</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
